### PR TITLE
Swagger fixes for v2

### PR DESCRIPTION
**Pull request by christroutner**
This PR represents the code changes highlighted in Issue #156.

### CODE REVIEW

File: src/routes/v2/dataRetrieval.ts [hunk 1]
```diff
@@ -777,6 +777,8 @@ async function pendingTransactions(
       return res.json({ error: "address can not be empty" })
     }
 
+    console.log(`address: ${address}`)
```
Comment (cgcardona) [inline src/routes/v2/dataRetrieval.ts]: Can you remove these two console.log statements/

Comment (christroutner) [reply src/routes/v2/dataRetrieval.ts]: Good catch.

**Comment by christroutner** [comment]
For some reason, your last comment didn't show up until after I hit the merge button. Here is a PR for that:
https://github.com/Bitcoin-com/rest.bitcoin.com/pull/167

**Comment by cgcardona** [comment]
:tada: This PR is included in version 1.14.1 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**